src/rust/src/x509/sct.rs

```diff
@@ -139,13 +157,18 @@ pub(crate) fn parse_scts(
         let log_id = sct_data.read_exact(32)?.try_into().unwrap();
         let timestamp = u64::from_be_bytes(sct_data.read_exact(8)?.try_into().unwrap());
         let _extensions = sct_data.read_length_prefixed()?;
-        let _sig_alg = sct_data.read_exact(2)?;
-        let _signature = sct_data.read_length_prefixed()?;
+        // TODO(alex): Convert the algorithms to enumerations
+        let hash_algorithm = u8::from_be_bytes(sct_data.read_exact(1)?.try_into().unwrap());
+        let signature_algorithm = u8::from_be_bytes(sct_data.read_exact(1)?.try_into().unwrap());
```

Comment:
Flagging for review: are enums for these available anywhere in Cryptography?

They should roughly be:

```py
class SCTHashAlgorithm(IntEnum):
    """
    Hash algorithms that are valid for SCTs.

    These are exactly the same as the HashAlgorithm enum in RFC 5246 (TLS 1.2).

    See: https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.1.4.1
    """

    NONE = 0
    MD5 = 1
    SHA1 = 2
    SHA224 = 3
    SHA256 = 4
    SHA384 = 5
    SHA512 = 6


class SCTSignatureAlgorithm(IntEnum):
    """
    Signature algorithms that are valid for SCTs.

    These are exactly the same as the SignatureAlgorithm enum in RFC 5246 (TLS 1.2).

    See: https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.1.4.1
    """

    ANONYMOUS = 0
    RSA = 1
    DSA = 2
    ECDSA = 3
```

Reply:
No, these enums don't exist anywhere ATM.

Reply:
Added enums with [22e1630](https://github.com/pyca/cryptography/pull/7207/commits/22e1630fc152756c3b0c827c060b38294bd4a6b3). These are technically in TLS 1.2, but 🤷